students/morgan/mailprog/mailprog/mailprog/tests/test_model.py

```diff
@@ -0,0 +1,76 @@
+from mailprog import dataModels, ui
+import os
+
+
+
+
+
+
+
+"""Donor class tests"""
+def test_create_donor():
+    bob = dataModels.Donor("Robert Dunbar", 1)
+    assert bob.name == "Robert Dunbar"
+    assert bob.donations == 1
```

Comment:
careful here -- if there is only one donation, `.donations` should probably still be a list. -- or an iterable of some sort.